<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DTM</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="Draw" name="Canvas" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Select shape:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>Hill</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Valley</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ridge</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Saddle</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Gentle Ridge</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Grid</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="generateTerrain">
       <property name="text">
        <string>Generate terrain shape</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="importPolygons">
       <property name="text">
        <string>Import polygons</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="createDelaunay">
       <property name="text">
        <string>Create Delaunay triangulation</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Lowest contour:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Highest contour:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="text">
        <string>500</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Contour interval:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="createContours">
       <property name="text">
        <string>Create contour lines</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Select the analyse:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_2">
       <item>
        <property name="text">
         <string>Slope</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Aspect</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="analyzeDTM">
       <property name="text">
        <string>Analyze DTM</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="Save">
       <property name="text">
        <string>Save picture</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearPoints">
       <property name="text">
        <string>Clear Points</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearDT">
       <property name="text">
        <string>Clear DT</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Draw</class>
   <extends>QWidget</extends>
   <header location="global">draw.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
